frameit/lib/frameit/device_types.rb

```diff
@@ -1,18 +1,98 @@
+require_relative 'device'
+require_relative 'module'
+require 'deliver/app_screenshot'
+
 module Frameit
   module Color
-    BLACK = "Space Gray"
-    SILVER = "Silver"
+    MATTE_BLACK = "Matte Black"
+    SPACE_GRAY = "Space Gray"
     ROSE_GOLD = "Rose Gold"
+    CLEARLY_WHITE = "Clearly White"
+    JUST_BLACK = "Just Black"
+    NOT_PINK = "Not Pink"
+    SILVER_TITANIUM = "Silver Titanium"
+    ARCTIC_SILVER = "Arctic Silver"
+    CORAL_BLUE = "Coral Blue"
+    MAPLE_GOLD = "Maple Gold"
+    MIDNIGHT_BLACK = "Midnight Black"
+    ORCHID_GRAY = "Orchid Gray"
+    BURGUNDY_RED = "Burgundy Red"
+    LILAC_PURPLE = "Lilac Purple"
+    SUNRISE_GOLD = "Sunrise Gold"
+    TITANIUM_GRAY = "Titanium Gray"
+    CORAL = "Coral"
+    BLACK = "Black"
```

Comment:
Ok, before Black was Space Gray.. we have to be aware of that.

Reply:
Yes I know. Before BLACK was Space Gray (or Matte Black in one device) and it was the default color. Now the default color is nil and each device has its own default color, for all Apple devices I assigned Space Gray or Matte Black if available. So it should be backward compatible, just if someone explicitly required "BLACK", it will show him a message like "Black not available, falling back to Space Gray".

Reply:
By the "nil" I mean if template_finder receives "nil" as the color, it will use the particular device's default color, i. e. space gray or matte black in Apple devices.

Reply:
Ok, defining the default color per device become possible as you created this new list of devices and their properties here for frameit, correct?

Automatic fallback is perfect and makes this a great and acceptable change.

Reply:
Yes, each device is assigned a color constant from this "enum" which serves as a fallback. This avoids a lot of if / else code. Thanks.